packages/server/src/api/routes/row.js

```diff
@@ -31,6 +31,11 @@ router
     usage,
     rowController.save
   )
+  .post(
```

Comment:
This will likely need updated with the resources system for RBAC which is now in master.